packages/table/src/Row.tsx

```diff
@@ -0,0 +1,312 @@
+import React, { useState, useEffect } from 'react';
+import { Transition } from 'react-transition-group';
+import IconButton from '@leafygreen-ui/icon-button';
+import Icon from '@leafygreen-ui/icon';
+import { isComponentType } from '@leafygreen-ui/lib';
+import { css, cx } from '@leafygreen-ui/emotion';
+import { uiColors } from '@leafygreen-ui/palette';
+import CheckboxCell from './CheckboxCell';
+import { useTableContext, Types, DataType } from './TableContext';
+
+const rowStyle = css`
+  cursor: pointer;
```

Comment:
This should only be pointer if we're in selectable mode right?

It's weird each row is a pointer when you can't click on anything.

Reply:
Good call